ansible/group_vars/sonic/sku-sensors-data.yml

```diff
@@ -956,7 +956,7 @@ sensors_checks:
       temp: []
     psu_skips: {}
 
-  Arista-7060CX-32S-C32:
+  Arista-7060-CX32S:
```

Comment:
Why change HW SKU?. We've it as Arista-7060CX-32S-C32

Reply:
Changed it back